msdm/tools/install_julia_pomdps.py

```diff
@@ -21,4 +22,4 @@ def install(julia="julia"):
     kw = dict()
     if len(args) == 1:
         kw['julia'] = args[0]
```

Comment:
Need to rename this to `julia_path` too for the argument to be passed through correctly